Makefile

```diff
@@ -69,16 +79,16 @@ codegen-odf-serde:
 	python $(ODF_SPEC_DIR)/tools/jsonschema_to_flatbuffers.py $(ODF_SPEC_DIR)/schemas > $(ODF_CRATE_DIR)/schemas/odf.fbs
 	flatc -o $(ODF_CRATE_DIR)/src/serde/flatbuffers --rust --gen-onefile $(ODF_CRATE_DIR)/schemas/odf.fbs
 	mv $(ODF_CRATE_DIR)/src/serde/flatbuffers/odf_generated.rs $(ODF_CRATE_DIR)/src/serde/flatbuffers/proxies_generated.rs
-	$(RUSTFMT) $(ODF_CRATE_DIR)/src/serde/flatbuffers/proxies_generated.rs
+	rustfmt $(ODF_CRATE_DIR)/src/serde/flatbuffers/proxies_generated.rs
```

Comment:
I think we should revert these changes.

Previous code was passing `--edition 2021` option to `rustfmt`. The default edition is still `2015` [according to the docs](https://rust-lang.github.io/rustfmt/?version=v1.7.0&search=#edition), and I distinctly remember having some formatting issues because of the edition mismatch.

Reply:
We have `rustfmt.toml` with specified `edition = "2021"`

Also, I double-checked that the file is read if `rustfmt` is run outside of `cargo fmt`